doc/modules/cassandra/pages/developing/cql/constraints.adoc

```diff
@@ -0,0 +1,53 @@
+= Constraints
```

Comment:
good this exists, it will probably need a little bit more examples, all what is possible etc, we should be as descriptive as possible.

Reply:
I expanded the documentation